### PR TITLE
update Talk basics docs: screenshot, markdown 

### PR DESCRIPTION
**Pull request by Antreesy**
##Markdown

image
image

## Reminder
image
image

### Updated screenshots
See files diff

### CODE REVIEW

File: user_manual/talk/talk_basics.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can create a direct, one-on-one chat by searching for the name of a user, a
 
 .. image:: images/chat-with-one-user.png
 
-If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``Create a new conversation``. 
+If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``Create a new conversation``.
```
Comment (DorraJaouad) [inline user_manual/talk/talk_basics.rst:21]: There is no ``⋮`` button anymore.

File: user_manual/talk/talk_basics.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ You can create a direct, one-on-one chat by searching for the name of a user, a
 
 .. image:: images/chat-with-one-user.png
 
-If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``Create a new conversation``.
```
Comment (DorraJaouad) [inline user_manual/talk/talk_basics.rst:21]: There is another one in join open conversation section ( line 43)

**Review by DorraJaouad — COMMENTED** [review]
Screenshots need to be update : 
- File upload ( caption was added)
- Other screenshots where the three dot menu is visible ( not critical though)

Also, I think we should add the guest name window to the doc

**Comment by Antreesy** [comment]
Yay, three months!

Please note that it's intended to be fully backported to stable27, where we didn't have guest window and caption.
Let's do another round later for stable28, based on milestones: https://github.com/nextcloud/spreed/milestones?state=closed

**Review by DorraJaouad — APPROVED** [review]

**Pull request merged**